code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -82,22 +82,24 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 /obj/item/device/pda/update_icon()
 	cut_overlays()
+	var/mutable_appearance/overlay = new()
+	overlay.pixel_x = overlays_x_offset
 	if(id)
-		var/image/I = image(icon_state = "id_overlay", pixel_x = overlays_x_offset)
-		add_overlay(I)
+		overlay.icon_state = "id_overlay"
+		add_overlay(new /mutable_appearance(overlay))

```

Comment:
Why do you make new one? It shouldn't matter here or is there something i'm missing ?